app/helpers/application_helper.rb

```diff
@@ -177,6 +177,10 @@ def email_link(type = :default, text: nil, additional_info: nil)
     mail_to email, text || email, additional_info
   end
 
+  def community_members_label
+    SiteConfig.community_member_label.pluralize
```

Comment:
although it's fine for now, the assumption that pluralization will always work for other communities member's designations might not always be true. Maybe the next cycle we can start thinking of using i18n or something :D

Reply:
Good call